configs/rag-dev-bootcamp-mar2024/aws.yaml

```diff
@@ -0,0 +1,10 @@
+head_node_type:
+  name: head_node_type
+  instance_type: g5.4xlarge
+worker_node_types:
```

Comment:
Do we need this if we are using serverless configs?

See endpoints OA template: https://github.com/anyscale/templates/blob/main/configs/endpoints_v2/aws.yaml

cc @shomilj @sofianhnaide